src/core/src/main.cpp

```diff
@@ -301,6 +301,10 @@ bool run(
         printf("Fail to load config file %s\n", config_file);
         return false;
     }
+    dwarn("load config file '%s' successfully", config_file);
+
+    // load plugged modules
+    load_all_modules(dsn_all.config);
```

Comment:
**load_all_modules** is put after pause because we also want to test the module initialization logic. Any particular reason to move it ahead (which disables the previous purpose)?